src/firefox/remote.js

```diff
@@ -0,0 +1,102 @@
+/* @flow */
+import {createLogger} from '../util/logger';
+import {WebExtError} from '../errors';
+import defaultFirefoxConnector from 'node-firefox-connect';
```

Comment:
`node-firefox-connect` seems to be missing from the dependencies currently specified in the `package.json`.